Makefile

```diff
@@ -1,11 +1,15 @@
 SKAFFOLD_VERSION=0.33.0
 CST_VERSION=1.8.0
 
-setup: .resize umask microk8s helm skaffold gitconfig ssh
+setup: .resize .profile umask microk8s helm skaffold gitconfig ssh
 
 .resize:
 	sh resize-volume.sh
 	touch .resize
+
+.profile:
+	grep -qxF 'sudo iptables -P FORWARD ACCEPT' ~/.profile || echo 'sudo iptables -P FORWARD ACCEPT' >> ~/.profile
```

Comment:
do we really want to forward EVERYTHING? or can we make this restrictive?

Reply:
Why not forward all the things?  Any specific suggestions?

Reply:
Updated to remove the other umask command/script after testing.